pkg/hardware/profile.go

```diff
@@ -76,6 +76,16 @@ func init() {
 		CPUArch: "x86_64",
 	}
 
+	profiles["hctl-0-2-0-0"] = Profile{
```

Comment:
The goal was to have the profile names be more meaningful (or generic?) than "use the second disk". Is there something about this hardware that we could use to describe it other than the HCTL entry?

Reply:
This is the first disk behind a per on teh del r720xd

Reply:
"dell2" ?  :-)

I figured all of these were just hacks until we add defining these dynamically via CRs, right?

Reply:
Maybe "dell-raid"?

Reply:
done!